spec/coverage_helper.rb

```diff
@@ -59,5 +59,8 @@ def expand_formatter(formatter)
     # Extra statistics to be placed in `rake stats`. We don't run that on CI, so coverage is not
     # important.
     add_filter '/lib/tasks/coursemology/stats_setup.rake'
+
+    # Rake task to seed dev database with course and assessment data.
+    add_filter '/lib/tasks/coursemology/seed.rake'
```

Comment:
Don't think we need this right? No point seeding the travis database as we create our own objects for different tests.